src/js/components/agencyV2/visualizations/totalObligationsOverTime/AgencyBudgetLine.jsx

```diff
@@ -66,27 +71,51 @@ const AgencyBudgetLine = ({
                     x: padding.left,
                     y: height - yScale(agencyBudget) - padding.top,
                     width: show ? xScale(todaysDate) : width - padding.left - padding.right,
-                    height: height - yScale(data[data.length - 1].obligated) - padding.bottom - padding.top
+                    height: height - yScale(data[data.length - 1].obligated) - padding.bottom - padding.top,
+                    balance: agencyBudget - data[data.length - 1].obligated,
+                    percentOfTotal: ((agencyBudget - data[data.length - 1].obligated) / agencyBudget) * 100
```

Comment:
We could use the `calculatePercentage` helper from `moneyFormatter` here

Reply:
Updated to use calculatePercentage!